styles.css

```diff
@@ -0,0 +1,109 @@
+*,
+*::after,
+*::before {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+body {
+  font-size: 1.6rem;
+}
+
+.roboto { font-family: 'Roboto', sans-serif; }
+.crete { font-family: 'Crete Round', serif; }
+.poppins { font-family: 'Poppins', sans-serif; }
+.inter { font-family: 'Inter', sans-serif; }
+
+.white { color: #fff; }
+.nBlue1 { color: #091e42; }
+.grey { color: #f4f5f7; }
+.orange { color: #ff6b00; }
+.blackish { color: #1c1a19; }
+.nBlue2 { color: #172b4d; }
+.white1 { color: #fff5e1; }
+
+.bcOrange { background-color: #ff6b00; }
+.bcGrey { background-color: #3c3a39; }
+.bcGrey1 { background-color: #3e3c3c; }
+.bcBlackish { background-color: #1c1a19; }
+
+.w400 { font-weight: 400; }
+.w500 { font-weight: 500; }
+.w700 { font-weight: 700; }
+
+.s15 { font-size: 0.938rem; }
+.s16 { font-size: 1rem; }
+.s17 { font-size: 1.063rem; }
+.s18 { font-size: 1.125rem; }
```

Comment:
- [ ] Kindly make your logo text larger to make it easier to read and so it can look as close as possible to the Figma design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/205102641-f43131a6-19c9-4a16-8c68-7932712ab5e0.png)  |  ![image](https://user-images.githubusercontent.com/74814780/205102579-06a308f3-9955-4436-96db-606dd606039e.png)

Reply:
Fixed.